packages/web/integrations/bridges/skip/types.ts

```diff
@@ -0,0 +1,194 @@
+export type Asset = {
+  denom: string;
+  chain_id: string;
+
+  origin_denom: string;
+  origin_chain_id: string;
+  trace: string;
+  is_cw20: boolean;
+
+  symbol?: string;
+  name?: string;
+  logo_uri?: string;
+  decimals?: number;
+  token_contract?: string;
+  description?: string;
+  coingecko_id?: string;
+  recommended_symbol?: string;
+};
+
+export type Chain = {
+  chain_name: string;
+  chain_id: string;
+  pfm_enabled: boolean;
+  cosmos_sdk_version: string;
+  supports_memo: boolean;
+  logo_uri?: string;
+  bech32_prefix: string;
+  chain_type: string;
+};
+
+export type RouteRequestBase = {
```

Comment:
What do you think if we prefix all the types in this file with `Skip`?

Reply:
Done